api/src/services/dwc-service.ts

```diff
@@ -90,4 +70,60 @@ export class DwCService extends DBService {
     // Apply patch operations
     return jsonpatch.applyPatch(jsonObject, patchOperations).newDocument;
   }
+
+  async decorateDWCASourceData(occurrenceSubmissionId: number): Promise<boolean> {
+    const occurrenceService = new OccurrenceService(this.connection);
+
+    const submission = await occurrenceService.getOccurrenceSubmission(occurrenceSubmissionId);
+    const jsonObject = submission.darwin_core_source;
+
+    const pathsToPatch = JSONPath({
+      path: '$..[verbatimCoordinates]^',
+      json: jsonObject,
+      resultType: 'all'
+    });
+
+    const patchOperations: Operation[] = [];
+
+    await Promise.all(
+      pathsToPatch.map(async (item: any) => {
+        // eslint-disable-next-line no-prototype-builtins
+        if (item.value.hasOwnProperty('decimalLatitude') && item.value.hasOwnProperty('decimalLongitude')) {
+          return;
+        }
```

Comment:
Do we need to also check if the `decimal Lat / Long` columns are not empty, in addition to the fields existing?